.gitignore

```diff
@@ -60,3 +60,4 @@ build/gccarch
 tests/_cache
 
 .php_cs.cache
+.idea
```

Comment:
Please do not use this ignore file to define platform specific files.
https://help.github.com/articles/ignoring-files/#create-a-global-gitignore